.travis.yml

```diff
@@ -38,14 +38,12 @@ before_install:
 
 install:
   - export GIT_FULL_HASH=`git rev-parse HEAD`
-  - conda create -n testenv pip nose pytest python=$TRAVIS_PYTHON_VERSION numpy=$NUMPY scipy scikit-image six coverage cython xraylib lmfit=0.8.3 netcdf4 flake8
+  - conda create -n testenv pip nose pytest python=$TRAVIS_PYTHON_VERSION numpy=$NUMPY scipy scikit-image six coverage cython xraylib lmfit=0.8.3 netcdf4 flake8 pyFAI codecov -c conda-forge
   - source activate testenv
-  - pip install pyFAI
   # # need to build_ext -i for the tests so that the .so is local to the source
   # # code.  We could also setup.py develop, but I'm not sure if that is any
   # # better
   - python setup.py install build_ext -i
-  - pip install codecov
```

Comment:
I'd prefer to keep pulling this from pip. Keeping the conda-forge recipe up to date will always lag behind the codecov folks uploading their own package to pypi and I'm not sure that we really gain anything from pulling codecov from conda-forge